pkg/controller/scylladbdatacenter/resource_test.go

```diff
@@ -741,6 +746,8 @@ exec /mnt/shared/scylla-operator sidecar \
 										}() + ` \
 --nodes-broadcast-address-type=ServiceClusterIP \
 --clients-broadcast-address-type=ServiceClusterIP \
+--developer-mode=true \
+--additional-scylladb-arguments="--log-to-stdout=1 --log-to-stderr=1" \
```

Comment:
this grows the flagset for something we plan to get rid of and is harder to escape - I think it will be better to use args that just propagate through, like

```
scylla-operator sidecar --loglevel=2 -- --developer-mode=1 --log-to-stdout=1 --log-to-stderr=1
```

and eventually became what we pass to scylladb directly

Reply:
Proposed format is ambigous to me, it's easy to confuse which parameters belongs to which binary. I prefer explicit name as it's clear where these should be used.

Reply:
This is a standard way to pass the args to another binary - literarly from `kubectl run --help`
```
  # Start the nginx pod using the default command, but use custom arguments (arg1 .. argN) for that command
  kubectl run nginx --image=nginx -- <arg1> <arg2> ... <argN>
```

Reply:
with `run` command it's more obvious, with `sidecar` it's not. Anyways, changed to use positional args.

Reply:
thanks, I think it will be even clearer when the sidecar (that's really a starter) becomes the starter